.github/workflows/pre-release.yml

```diff
@@ -194,18 +194,51 @@ jobs:
           username: ${{ secrets.REGISTRY_USER }}
           password: ${{ secrets.REGISTRY_PASSWORD }}
 
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: "ghcr.io"
+          username: "keptn-bot"
+          password: ${{ secrets.KEPTN_BOT_TOKEN }}
```

Comment:
can we not use the included `GITHUB_TOKEN` here?

Reply:
I want the docker images to be pushed by the keptn-bot user and not the user that runs the release pipeline which would be the result if we use the `GITHUB_TOKEN` i think